packages/homepage/src/components/Toast/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import { Toastcontainer } from './elements';
+
+const Privacy = () => (
```

Comment:
Make the component a `FunctionComponent` & export it as a named export in order to be the same as all our new components.

```suggestion
export const Privacy: FunctionComponent = () => (
```